app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -150,5 +150,7 @@ def destroy_mapped_tenants
         source_tenant.destroy
       end
     end
+
+    define_method(:allow_nonunique_endpoint_url?) { true }
```

Comment:
Can you also add this to the top level `ExtManagementSystem` class (returning false obviously) so it is explicit rather than relying on the method not being there.

Reply:
ops, comment https://github.com/ManageIQ/manageiq/pull/18298#discussion_r243341456 should be here :)

Reply:
done